sunpy/net/vso/table_response.py

```diff
@@ -83,13 +83,12 @@ def from_zeep_response(cls, response, *, client, _sort=True):
                         # doesn't recognise as a unit, so fix it.
                         waveunit = value['waveunit']
                         waveunit = 'keV' if waveunit == 'kev' else waveunit
-
                         row["Wavelength"] = None
-                        if value['wavemin'] is not None and value['wavemax'] is not None:
+                        # VSO added "N/A" as potential value for wavemin and wavemax
+                        if value['wavemin'] not in [None, "N/A"] and value['wavemax'] not in [None, "N/A"]:
```

Comment:
I would like to get word from the VSO that this is a permanent change before we do this.

Reply:
@ejm4567 What are your thoughts on this change?

Reply:
I've put a temp fix in the VSO Core for Lyra so you shouldn't see that error in SunPy again. It turns out the N/A is what is being returned by the TAP query in wavelength_range for Lyra. Swap populates that field with actual wavelengths. I need to look in detail at what other fields Lyra may be populating wherein I can then map correct, accurate wavelengths for Lyra. The fix would ideally be in the VSO wherein the query results are first processed.

Reply:
Thanks! So it should be safe for me to revert this now?